isis/src/base/apps/ringsautomos/ringsautomos.cpp

```diff
@@ -0,0 +1,116 @@
+#include "ProcessMapMosaic.h"
+#include "FileList.h"
+#include "IException.h"
+#include "SpecialPixel.h"
+#include "RingPlaneProjection.h"
+#include "ProjectionFactory.h"
+
+#include "ringsautomos.h"
+
+using namespace std;
+
+namespace Isis {
+
+  void ringsautomos(UserInterface &ui, Pvl *log){
+    FileList list;
+
+    // Get the list of cubes to mosaic
+    list.read(FileName(ui.GetFileName("FROMLIST")));
+    // Redundant test.  Already checked in FileList
+    // if(list.size() < 1) {
+    //   QString msg = "The list file [" + ui.GetFileName("FROMLIST") +"does not contain any data";
+    //   throw IException(IException::User, msg, _FILEINFO_);
+    // }
+
+    fstream os;
+    bool olistFlag = false;
+    if (ui.WasEntered("TOLIST")){
+      QString olist = ui.GetFileName("TOLIST");
+      olistFlag = true;
+      os.open(olist.toLatin1().data(), std::ios::out);
+    }
+
+    ProcessMapMosaic m;
+
+    // Set the create flag-mosaic is always created in ringsautomos
+    m.SetCreateFlag(true);
+
+    // Get the Track Flag
+    bool bTrack = ui.GetBoolean("TRACK");
+    m.SetTrackFlag(bTrack);
+
+    ProcessMosaic::ImageOverlay overlay = ProcessMosaic::StringToOverlay(
+        ui.GetString("PRIORITY"));
+
+    if (overlay == ProcessMapMosaic::UseBandPlacementCriteria) {
```

Comment:
Can we change this to use `ProcessMosaic` instead of `ProcessMapMosaic`? The enum is defined in `ProcessMap.h` but is inherited in `ProcessMapMosaic.h` Just clears things up so people don't think there are two different enums.

Reply:
Looks like this may have been a problem? I made this change and built locally before the app conversion and things worked. Did an include get removed?